docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  home-assistant:
+    image: homeassistant/home-assistant
+    container_name: home-assistant
+    restart: unless-stopped
+    network_mode: host
```

Comment:
I'm not sure about this. With `host` network mode you effectively loose most of the security benefits of using docker. I do understand that `discovery:` doesn't work too well without host network mode.

Maybe let's think about the target audience for this file: Is it users that want a) an easy setup process or b) to use docker for its containerization features? I think the normal piolp installer already serves a) target audience quite well. For b) `expose` instead of `network_mode` would be better.